src/common/types/include/types.h

```diff
@@ -123,5 +125,9 @@ string getRelDirectionAsString(RelDirection relDirection);
 
 enum class DBFileType : uint8_t { ORIGINAL = 0, WAL_VERSION = 1 };
 
+spdlog::level::level_enum convertStrToLevelEnum(string loggingLevel);
```

Comment:
Move the conversion for log level out of Types as some other utility functions.

Reply:
Moved this function to logging_level_utils.h